backend/src/test/java/net/wodie/backend/controller/BtoControllerTest.java

```diff
@@ -0,0 +1,71 @@
+package net.wodie.backend.controller;
+
+import net.wodie.backend.model.BtoItem;
+import net.wodie.backend.repository.BtoRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class BtoControllerTest {
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private WebTestClient testClient;
+
+    @Autowired
+    private BtoRepository btoRepository;
+
+    @BeforeEach
+    public void cleanUp() {
+        btoRepository.deleteAll();
+    }
+
+    @Test
+    void getAllBtoItems_AllOk() {
+        //GIVEN
+        BtoItem item1 = new BtoItem("1", "t01c1o01","action","Actionpoint 01",
+                "First Day","<p>Testeintrag</p>","open","U11",
+                new String[]{"U12","U13"}, new String[]{"U15"});
```

Comment:
Sicherlich Geschmackssache, aber bei so vielen Parametern des Konstruktors ziehe ich die builder-Schreibweise vor, damit deutlich wird, welcher Wert in welches Feld gespeichert wird.

Reply:
Der Test wurde auf Builder umgebaut